hyperspy/learn/mva.py

```diff
@@ -958,6 +959,170 @@ def plot_explained_variance_ratio(self, n=50, log=True):
             ax.semilogy()
         return ax
 
+    def scree_plot(self, n=50, log=True, cutoff=0.01, signal_num=0,
+                   xaxis=None, signal_fmt=None, noise_fmt=None,
+                   axes_titles=None, fig=None, ax=None,
+                   **kwargs):
+        """Plot the decomposition explained variance ratio vs index number
+        (Scree Plot). This method is an alternative to
+        `plot_explained_variance_ratio`
+
+        Parameters
+        ----------
+        n : int
+            Number of components to plot.
+        log : bool
+            If True, the y axis uses a log scale.
+        cutoff : float or None
+            y-position at which to draw a line showing the cutoff between
+            signal and noise
+        signal_num : int
+            Number of components to highlight as important
+            (i.e. the signal components as opposed to noise).
+            Only used if `cutoff` is None, will be automatically determined
+            otherwise
+        xaxis : dict or None
```

Comment:
Why not splitting this in two kwargs e.g. `xaxis_type`, `xaxis_labeling`? I think that it is more orthodox and user friendly this way.